pre_commit/xargs.py

```diff
@@ -24,13 +24,11 @@
 
 
 def cpu_count() -> int:
-    try:
-        # On systems that support it, this will return a more accurate count of
-        # usable CPUs for the current process, which will take into account
-        # cgroup limits
+    # On systems that support it, this will return a more accurate count of
+    # usable CPUs for the current process, which will take into account
+    # cgroup limits
+    if hasattr(os, 'sched_getaffinity'):
         return len(os.sched_getaffinity(0))
```

Comment:
this makes this ever-so-slightly worse in that it grabs the attribute twice in the "normal" path